.travis.yml

```diff
@@ -24,6 +24,6 @@ script:
     - export PATH="/tmp/dummy_bin:$PATH"
     # Setup Avocado-vt for functional tests
     - make install
-    - AVOCADO_LOG_DEBUG=yes avocado vt-bootstrap --yes-to-all
+    - AVOCADO_LOG_DEBUG=yes avocado vt-bootstrap --vt-skip-verify-download-assets --yes-to-all
```

Comment:
tbh I'd probable prefer keeping the `--no-download` usage and add `--no-providers-download`, because this changes all the existing scripts out there (my CI including)